### ISSUE TITLE
Bounding box predictions misaligned

### ISSUE THREAD

**Issue by ProfessorHT0** [httle/ARS-DETR#27]
Hello,

First of all, thank you for this great project and for making the code publicly available.

I am currently testing the model dn_arw_arm_arcsl_rdetr_r50_dota on the DOTA-v1 validation and test datasets using the following command:
`python tools/test.py configs/ars_detr/dn_arw_arm_arcsl_rdetr_r50_dota.py checkpoints/dn_arw_arm_arcsl_rdetr.pth --show-dir Dota/vis`

However, the predicted bounding boxes appear to be significantly misaligned with the objects in the images. The detections are not positioned correctly and do not correspond well to the visible targets.

I am testing on:

- DOTA-v1 validation set
- DOTA-v1 test set
- Default configuration provided in the repository
- No modifications to the pretrained checkpoint

Please find several example visualizations illustrating the issue at the following link: [ARS-DETR](https://mega.nz/folder/XigRxAbK#P8C15bullZ-c_XiIlYYqMQ)

Is there any known issue related to bounding box decoding or rescaling during inference?

Thank you very much for your time and help.

Best regards,
